cmd/template/nodepool/provider/templates/azure/template.go

```diff
@@ -17,10 +17,10 @@ var kubeadmConfig string
 // GetTemplate merges all .tmpl files.
 func GetTemplate() string {
 	return strings.Join([]string{
+		// Adds a separator at the beginning of the joined template for easier joinability of cluster and node pool templates.
```

Comment:
Cluster template does not end with a separator so I ensure I have a separator at the beginning of the template to be able to do:

```
kubectl-gs template cluster ... >cluster.yaml
kubectl-gs template nodepool ... >>cluster.yaml
```

and get a valid list of templates.